src/lib.rs

```diff
@@ -0,0 +1,33 @@
+mod bevy_setup;
+mod core;
+#[cfg(feature = "dev")]
+mod dev;
+mod ui;
+
+use bevy::prelude::*;
+
+pub struct GamePlugin;
```

Comment:
Wondering if this should be `pub fn plugin` for consistency, and `.add_plugins(bevy_template::plugin)` in `main.rs`.

Reply:
I think it's special enough to be this way

Reply:
Understandable.